engine/src/org/pentaho/di/trans/steps/scriptvalues_mod/ScriptValuesAddedFunctions.java

```diff
@@ -2641,12 +2666,12 @@ public static Boolean isMailValid( Context actualContext, Scriptable actualObjec
           return Boolean.FALSE;
         }
 
-        retval = Boolean.TRUE;
+        isValid = Boolean.TRUE;
 
       } catch ( Exception e ) {
         throw Context.reportRuntimeError( "Error in isMailValid function: " + e.getMessage() );
       }
-      return retval;
+      return isValid;
```

Comment:
You are right, it can.

But It's just a place where I refactored a variable name to camel case style (with no intention to change logic)
Actually there are bunch of such drawbacks in this class (it's ~ 3 k. lines length), so I decided just to make sure that my code is safe, and refactored a few things that strike the eyes without changing any logic.

So, I highly recomend to refactore this class and to make it more safe, but I would rather do this in context of the hole class, but not just in this method. But I need time and aggrement for this :)